app/src/store/user.ts

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+import type { PayloadAction } from '@reduxjs/toolkit';
+
+interface UserState {
+  accessToken: string;
+}
+
```

Comment:
Ideally, this is where I'd like all the authentication functionality to behave, i.e. calls to Auth0, getting Auth0 tokens, etc.